scripts/get_used_routes.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+
```

Comment:
par défaut, un script bash continue de s'éxécuter s'il y a une erreur

c'est souvent un comportement qu'on veut éviter, surtout dans une pipeline

pour changer ce comportement et s'arréter dès la première erreur rencontrée (et ajouter d'ajouter d'autres renforcement du meme style)

```suggestion
set -o errexit -o nounset -o pipefail
```